public-ui/src/publicGenerated/.gitignore

```diff
@@ -0,0 +1,4 @@
+wwwroot/*.js
```

Comment:
In the api dir, we don't check in this generated code. Please update this gitignore rule: https://github.com/all-of-us/workbench/blob/master/.gitignore#L46